docs/community/team.md

```diff
@@ -46,6 +46,12 @@ napari is a consensus-based community project. Anyone with an interest in the pr
 - [Nicholas Sofroniew](https://github.com/napari/napari/commits?author=sofroniewn) - [@sofroniewn](https://github.com/sofroniewn)
 - [Talley Lambert](https://github.com/napari/napari/commits?author=tlambert03) - [@tlambert03](https://github.com/tlambert03)
 
+# Contracted Roles
```

Comment:
Why not simplify the title to "Team roles"

And add a lead-in sentence:

To best serve napari's development, operations, and community, several team roles have contracts to focus on these areas: